guesttemplates/blank_template.py

```diff
@@ -23,7 +20,7 @@ def amount_to_int(amt):
 
 def get_bool_key(d, key, default):
     v = d.get(key, default)
-    if isinstance(v, basestring):
+    if isinstance(v, str):
         v = v.lower() in ('t', 'true', 'y', 'yes', '1')
```

Comment:
I like the idea and it would be perfect if it would remove all uses of basestring, but there's a second one for which I think it wouldn't be as elegant once compatibility with python2 is removed. So I suggest keeping the current way: first import `basestring` from `past` as done by `futurize`, then replace it with `str` once we don't need python2 anymore (that is, two commits after).

Or, if we want to make it really simpler, I could now squash everything into one commit and we wouldn't see the somewhat ugly transition that is due to the code being compatible with both python 2 and 3 .